web3.js/src/publickey.js

```diff
@@ -2,8 +2,12 @@
 
 import BN from 'bn.js';
 import bs58 from 'bs58';
+import nacl from 'tweetnacl';
 import {sha256} from 'crypto-hash';
 
+//$FlowFixMe
+let naclLowLevel = nacl.lowlevel;
```

Comment:
@mvines Hack to get at nacl's lowlevel functions so that we don't have to duplicate them in our codebase. Yuk, any suggestions?

Reply:
Can you also file an issue upgrade in nacl so they can poke fun at us, I mean, become aware of our need and maybe they'll have a nicer solution (or would be willing to expose one)